domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/transfers/Transfer.java

```diff
@@ -163,11 +163,11 @@ public TransferBuilder minTransferTime(@Nullable final Integer minTransferTime)
                             "transfer_type", fromStopId + ";" + toStopId,
                             originalTransferTypeInteger));
                 }
-                if (minTransferTime != null && minTransferTime < 1) {
-                    // here minTransferTime threshold should be user configurable
-                    noticeCollection.add(new IntegerFieldValueOutOfRangeNotice("transfers.txt",
-                            "min_transfer_time", fromStopId + ";" + toStopId, 0,
-                            Integer.MAX_VALUE, minTransferTime));
+                if (minTransferTime != null && ((minTransferTime < minTransferTimeLowerBound) ||
+                        (minTransferTime > minTransferTimeUpperBound))) {
```

Comment:
Wondering if this should be done here or at a later stage in its own use case. What were the reasons that led to have this implicit rule checking at entity creation time?

Reply:
As discussed, this will be implemented in a distinct use case.